packages/data-point/lib/entity-types/entity-hash/resolve.js

```diff
@@ -5,17 +5,15 @@ const Promise = require('bluebird')
 const utils = require('../../utils')
 
 function resolveMapKeys (accumulator, reducer, resolveTransform) {
-  const transform = { reducers: [reducer] }
```

Comment:
doing `const transform = { reducers: [reducer] }` was a hack that's no longer necessary. It was only done that way because reducers were always expected to be wrapped in TransformExpressions.

Reply:
and one more 👍 for commenting on why u removed it, thanks!